openstack/keystone/templates/etc/_keystone.conf.tpl

```diff
@@ -66,9 +66,13 @@ access_token_duration = {{ .Values.api.oauth1.access_token_duration | default "0
 {{- end }}
 
 [cache]
+{{- if and .Values.memcached.auth.username .Values.memcached.auth.password }}
+backend = dogpile.cache.bmemcached
+{{- else }}
 backend = dogpile.cache.memcached
+{{- end }}
 {{- if .Values.global.is_global_region }}
-memcached_servers = "{{ include "helm-toolkit.utils.joinListWithComma" .Values.memcached.server_ips_ports }}"
```

Comment:
That was actually the setting name for previous OpenStack releases, for instance: https://docs.openstack.org/keystone/xena/admin/configuration.html#keystonemiddleware-configuration-options-that-affect-performance